tests/Behat/Context/Ui/CreditMemoContext.php

```diff
@@ -136,16 +136,18 @@ public function itShouldContainProductWithNetValueTaxAmountAndGrossValueInCurren
     }
 
     /**
-     * @Then it should contain :quantity :shipmentName shipment with :grossValue gross value in :currencyCode currency
+     * @Then it should contain :quantity :productName shipment(s) with :netValue net value, :taxAmount tax amount and :grossValue gross value in :currencyCode currency
```

Comment:
Why did you change `shipmentName` to `productName` here?

Reply:
too many copy-past :( ofc it should be `shipmentName`)